Source/ORTS.Common/Conversions.cs

```diff
@@ -913,5 +932,33 @@ public static string FormatApproximateTime(double clockTimeSeconds)
 
             return string.Format("{0:D2}:{1:D2}", hour, minute);
         }
+
+
+        /// <summary>
+        /// Calculate and format horsepower per ton for consist, rounded to one decimal.
+        /// </summary>
+        /// <param name="consistPowerW"></param>
+        /// <param name="consistMassKG"></param>
+        /// <returns>horsepower-per-ton formated to one decimal.</returns>
+        //TODO: implement UK and metric version
+        public static string FormatHPT(float consistPowerW, float consistMassKG)
+        {
+            var hpt = consistMassKG > 0 ? W.ToHp(consistPowerW) / Kg.ToTUS(consistMassKG) : 0;
+            return string.Format("{0:0.0}", hpt);
+        }
+
+
+        /// <summary>
+        /// Calculate and format tons per operative brake for consist, rounded to an integer.
+        /// </summary>
+        /// <param name="consistMassKG"></param>
+        /// <param name="consistNumOpBrakes"></param>
+        /// <returns>tons-per-operative-brake formated to an integer.</returns>
+        //TODO: implement UK and metric version
+        public static string FormatTPOB(float consistMassKG, float consistNumOpBrakes)
+        {
+            var tpob = consistNumOpBrakes > 0 ? Kg.ToTUS(consistMassKG) / consistNumOpBrakes : 0;
+            return string.Format("{0:0}", tpob);
+        }
```

Comment:
I would prefer we skip these two formats; they haven't been needed anywhere else to date, so I'm not sure on the merits of adding them just for an unofficial project (all `Contrib` code is ~unofficial).

The lack of metric/imperial/etc. implementation is also an issue.

Let's move them into the Content Manager project for now, and they can always be moved here in the future if we need them.

Reply:
Done.